src/composed/message/types.rs

```diff
@@ -1061,25 +1061,28 @@ mod tests {
 
         let lit_msg = Message::new_literal("hello.txt", "hello world\n");
         let compressed_msg = lit_msg.compress(CompressionAlgorithm::ZLIB).unwrap();
-        for _ in 0..1000 {
-            let encrypted = compressed_msg
-                .encrypt_to_keys_seipdv2(
-                    &mut rng,
-                    SymmetricKeyAlgorithm::AES128,
-                    AeadAlgorithm::Ocb,
-                    0x06,
-                    &[&pkey][..],
-                )
-                .unwrap();
 
-            let armored = encrypted.to_armored_bytes(None.into()).unwrap();
-            fs::write("./message-x25519.asc", &armored).unwrap();
+        for aead in [AeadAlgorithm::Ocb, AeadAlgorithm::Eax, AeadAlgorithm::Gcm] {
+            for sym in [
+                SymmetricKeyAlgorithm::AES128,
+                // SymmetricKeyAlgorithm::AES192, // TODO: not currently supported
```

Comment:
That also surprised me, when i added this test :sweat_smile: 

AES192 was not implemented in https://github.com/rpgp/rpgp/blob/main/src/crypto/aead.rs.
When I wrote this test, I thought this AES192 can't be supported without changed to RustCrypto. However, now that I took a more serious look, I see that this is incorrect.

I added AES192 support in 9f2cb1174a0c902ed7790d080e6eed31abe47ed8, and enabled these tests.

I'm thinking of producing SEIPDv2 encrypted test-messages for all permutations of Eax, Ocb, Gcm x AES128/192/256. And I'm thinking it might be good to make those the payload big enough that the messages will be broken into at least two chunks (given a small chunk size that i can set when generating those messages). Does that sound reasonable/worthwhile?